#10 Figure with caption.

adamf321 opened the pull request.

mitogh commented on dist/_patterns/molecules/media/_figure-with-caption.scss:
```diff
@@ -0,0 +1,6 @@
+@import "~params/colors";
```
Indentation should be 2 spaces, can we run the CI to make it automatic checks instead.

adamf321 commented:
I added the JS and SASS lints scripts. I've changed the names from "ci" to "lint" as this better describes what the script does (e.g. `npm run js:lint`).
